doc/main/reference/scalable_memory_pools/malloc_replacement_log.rst

```diff
@@ -1,7 +1,7 @@
 .. _malloc_replacement_log:
 
-oneTBB_malloc_replacement_log Function
-======================================
+TBB_malloc_replacement_log
+==========================
```

Comment:
The "Function" was appropriate in the page header. Sorry if my comment was misleading in this regard.